### ISSUE TITLE
make crash

### ISSUE THREAD

**Issue by Yexeed** [MCMrARM/mcpelauncher-linux#243]
[  3%] Building CXX object CMakeFiles/mcpelauncher.dir/src/main.cpp.o
In file included from /home/yexeed/Рабочий стол/mcpe/src/main.cpp:38:0:
/home/yexeed/desktop/mcpe/src/extract.h:4:17: fatal error: zip.h: No such file or directory
compilation terminated.
CMakeFiles/mcpelauncher.dir/build.make:446: recipe error «CMakeFiles/mcpelauncher.dir/src/main.cpp.o»
make[2]: *** [CMakeFiles/mcpelauncher.dir/src/main.cpp.o] Error 1
CMakeFiles/Makefile2:67: ошибка выполнения рецепта для цели «CMakeFiles/mcpelauncher.dir/all»
make[1]: *** [CMakeFiles/mcpelauncher.dir/all] Ошибка 2
Makefile:127: ошибка выполнения рецепта для цели «all»

**Comment by guppy42** [comment]
Please install dependencies...

**Comment by Yexeed** [comment]
I did

**Comment by guppy42** [comment]
Maybe try reading the README ?

**Comment by Yexeed** [comment]
@guppy42 I installed all dependencies from readme, actually.

**Comment by Yexeed** [comment]
UPD: maybe this will help? (https://github.com/MCMrARM/mcpelauncher-linux/issues/205#issuecomment-334505826)

**Comment by MCMrARM** [comment]
>Maybe try reading the README
Yeah, not the best tip currently since it's not updated. Let me actually fix it up.

**Comment by r7vmc** [comment]
to make it easier for you
build on docker container
using this [Dockerfile](https://github.com/Irioth/mcpelauncher-docker)

**Comment by r7vmc** [comment]
bw You are missing ```libzip-dev:i368```

**Comment by elvaalxv** [comment]
Don't worry about that! 
I have recorded a video to guide you how to install mcpelauncher with the easiest way. Follow the the video, I sure that you will install mcpelauncher successfully. 
Check it! 
https://youtu.be/KsuS9IbMaVI
